Running `npm run env:start` fails with PHP 7.4

### Bug Description

When running `npm run env:start`, I'm getting an error:

```
pecl/xdebug requires PHP (version >= 8.0.0, version <= 8.2.99), installed version is 7.4.33
```

Full output:

```
$ npm run env:start

> amp-wp@ env:start /home/westonruter/repos/amp
> wp-env start --xdebug

✖ Error while running docker-compose command.
Step 1/7 : FROM wordpress:php7.4
 ---> d7568c9796d9
Step 2/7 : RUN apt-get -qy install $PHPIZE_DEPS && touch /usr/local/etc/php/php.ini
 ---> Using cache
 ---> 7d1c954c1770
Step 3/7 : RUN if [ -z "$(pecl list | grep xdebug)" ] ; then pecl install xdebug ; fi
 ---> Running in a4171cdf0110
pecl/xdebug requires PHP (version >= 8.0.0, version <= 8.2.99), installed version is 7.4.33
No valid packages found
install failed
Building wordpress
Service 'wordpress' failed to build: The command '/bin/sh -c if [ -z "$(pecl list | grep xdebug)" ] ; then pecl install xdebug ; fi' returned a non-zero code: 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amp-wp@ env:start: `wp-env start --xdebug`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the amp-wp@ env:start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/westonruter/.npm/_logs/2023-04-05T20_46_18_749Z-debug.log
```

### Expected Behaviour

Start without error.

We may need to bump the `phpVersion` to `8.0` in `.wp-env.json`.

I tried manually making this change, and `npm run env:start` succeeded.

### Screenshots

_No response_

### PHP Version

7.4

### Plugin Version

develop

### AMP plugin template mode

Standard, Transitional, Reader

### WordPress Version

_No response_

### Site Health

_No response_

### Gutenberg Version

_No response_

### OS(s) Affected

_No response_

### Browser(s) Affected

_No response_

### Device(s) Affected

_No response_

### Acceptance Criteria

_No response_

### Implementation Brief

_No response_

### QA Testing Instructions

_No response_

### Demo

_No response_

### Changelog Entry

_No response_

--- a/.wp-env.json
+++ b/.wp-env.json
@@ -1,6 +1,6 @@
 {
   "core": "WordPress/WordPress#master",
-  "phpVersion": "7.4",
+  "phpVersion": "8.0",
   "env": {
     "development": {
       "plugins": [

